ebib-utils.el

```diff
@@ -2376,25 +2392,19 @@ year can be extracted from DATE, return nil."
   (ebib-clean-TeX-markup (ebib-get-field-value field key db "" 'unbraced 'xref)))
 
 (defun ebib-clean-TeX-markup (string)
-  "Return STRING without TeX markup."
+  "Return STRING without TeX markup.
+
+Any substring matching the car of a cell in
+`ebib-TeX-markup-replace-alist' is replaced with the
+correspnoding cdr, if the cdr is a string, or with the result of
```

Comment:
I'd put `"if the cdr is a string"` in parentheses, because the equivalent remark one line below is, as well. (Or don't put parentheses around either.)

Reply:
Done